lib/util/array.ts

```diff
@@ -0,0 +1,8 @@
+import is from '@sindresorhus/is';
+
+export function coerceArray(input: unknown): any[] {
```

Comment:
yes, but the type is not represent the real type on runtime. most of the time it's a real type, but at runtime it can be a wrong type and we can't solve this at compile time. so we should infer the expected type here, which is the input type.

we can later use `z` lib to make runtime constraints, like @zharinov recently started on.